File: hw/santorum/u2/hw19Inheritance/Python/model/location.py
class Location:
    def __init__(self, xCoordinate, yCoordinate):
        self.xCoordinate = xCoordinate
        self.yCoordinate = yCoordinate

    def __str__(self):
        return f"Location(x={self.xCoordinate}, y={self.yCoordinate})"

File: hw/aymacana/u2/hw19Inheritance/FarmSystemJS/Location.js
class Location {
    constructor(xCoordinate, yCoordinate) {
        this._xCoordinate = xCoordinate;
        this._yCoordinate = yCoordinate;
    }

    get xCoordinate() { return this._xCoordinate; }
    get yCoordinate() { return this._yCoordinate; }

    set xCoordinate(xCoordinate) { this._xCoordinate = xCoordinate; }
    set yCoordinate(yCoordinate) { this._yCoordinate = yCoordinate; }

    toString() {
        return `Location{xCoordinate=${this._xCoordinate}, yCoordinate=${this._yCoordinate}}`;
    }
}

module.exports = { Location };